paket.dependencies

```diff
@@ -12,4 +12,12 @@ nuget xunit.runner.visualstudio ~> 2 version_in_path: false
 nuget Unquote
 
 # Example container to illustrate integration
-nuget Autofac
\ No newline at end of file
+nuget Autofac
+
+group build
+framework: auto-detect
```

Comment:
indent please (by 4 spaces like in rest of repo)

Reply:
done